src/CMakeLists.txt

```diff
@@ -10,70 +10,10 @@ include_directories("classlibnative/cryptography")
 include_directories("classlibnative/inc")
 include_directories("${GENERATED_INCLUDE_DIR}")
 
-# The Following Logic is used to wire up Build dependencies for Generated files in Event Logging 
-# ClrEtwAll.man                  - Event Schema
-# ClrEtwAllMeta.lst              - MetaData list [provided to ensure Windows Desktop is not broken]
-# genXplatEventing.py            - has the core logic for parsing Event Schema
-# genWinEtw.py                   - Uses genXplatEventing to generate Windows Specific ETW Files
-# clretwallmain.h and etmdummy.h - Provides the Event Logging Functionality to the VM
-# clrxplatevents.h               - Used by clretwallmain.h on Non Windows platform
-# ClrEtwAll.h                    - Used by clretwallmain.h on  Windows 
-# ClrEtwAll.rc                   - Used by src/dlls/clretwrc/clretrw.rc on Windows
-
-set (ScriptGeneratedEventFiles
-      ${GENERATED_INCLUDE_DIR}/clretwallmain.h
-      ${GENERATED_INCLUDE_DIR}/etmdummy.h
-)
-set (GeneratedEventFiles)
+if(NOT CLR_CMAKE_PLATFORM_UNIX)
```

Comment:
Since this is ETW specific stuff, it's probably better to use if(WIN32) instead.

Reply:
Does WIN32 catch 64-bit Windows?